cmd/posix.go

```diff
@@ -209,6 +209,16 @@ func isDirEmpty(dirname string) bool {
 	return true
 }
 
+func testFSCapabilities(path string) error {
```

Comment:
This function checks only for DIRECTIO support but `testFSCapabilities` hides what is being checked for. We could call this function `fsCheckDirectIO` or something to that effect.